int32 nPacman
pos pacmanPos
